### DISCUSSION TITLE
"allow-rename off" is not working

### DISCUSSION THREAD

**Discussion by nicolas-g** [catppuccin/tmux#309] [Help] [answered]
I have both `set -g allow-rename off` and `setw -g allow-rename off` in my tmux config but unfortunately since I started using catppuccin the window title always match the directory I'm in.

I would like the window title to display the name I manually name it

**Comment by vdbe** [comment, marked as answer]
See #227

**Reply by nicolas-g** [reply]
thank you @vdbe , worked like a charm !
